index.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script
+      src="https://kit.fontawesome.com/4993eda318.js"
+      crossorigin="anonymous"
+    ></script>
+    <link href="./css/main.css" rel="stylesheet" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap"
+      rel="stylesheet"
+    />
+    <title>Blog Minimo</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <div class="row">
+          <div class="header__logo col-sm-5 col-md-4 col-lg-4">
+            <a href="#"><h1>Minimo</h1></a>
+          </div>
+
+          <div class="header__hamburger col-sm-5 col-md-6 col-lg-7">
```

Comment:
To menu zachowuje się dziwnie:

![Screenshot 2019-10-27 at 18 00 36](https://user-images.githubusercontent.com/20945507/67638353-d49f8700-f8e3-11e9-903e-92b78e191227.png)

Wygląda jakby było nienaturalnie przesunięte od prawej. Do tego jeśli zostawisz je otwarte na mobile'u to po zmianie szerokości ekranu do tablet / desktop dzieje się coś takiego:

![Screenshot 2019-10-27 at 18 02 40](https://user-images.githubusercontent.com/20945507/67638372-0dd7f700-f8e4-11e9-967d-e9aef5f11e9f.png)